lib/model/compose.dart

```diff
@@ -5,6 +5,31 @@ import 'internal_link.dart';
 import 'narrow.dart';
 import 'store.dart';
 
+/// The available user wildcard mention options,
+/// known to the server as [canonicalString].
+///
+/// There isn't an API doc for these yet, but see the values in the Help Center:
+///   https://zulip.com/help/mention-a-user-or-group
+///   https://zulip.com/help/dm-mention-alert-notifications
+enum WildcardMentionOption {
+  all(canonicalString: 'all'),
+  everyone(canonicalString: 'everyone'),
+  channel(canonicalString: 'channel'),
+  // Deprecated in FL 247. Empirically, current servers (FL 330)
+  // still parse "@**stream**" in messages though.
+  stream(canonicalString: 'stream'),
+  topic(canonicalString: 'topic'); // New in FL 224.
+
+  /// The string identifying this option (e.g. "all" as in "@**all**").
+  // If servers want to localize this, we can compute it
+  // dynamically from server data instead of hard-coding it.
+  const WildcardMentionOption({required this.canonicalString});
```

Comment:
I think this dartdoc is meant to go on the `canonicalString` field rather than the constructor?

…Ah I see this came from some code I proposed 😅